config/kubernetes/sig-scheduling/teams.yaml

```diff
@@ -2,170 +2,82 @@ teams:
   sig-scheduling-api-reviews:
     description: ""
     members:
-    - aveshagarwal
-    - bsalamat
-    - ConnorDoyle
-    - davidopp
-    - derekwaynecarr
-    - hongchaodeng
-    - ingvagabund
-    - jayunit100
-    - k82cn
-    - kevin-wangzefeng
-    - pigmej
-    - quinton-hoole
-    - resouer
-    - rrati
-    - timothysc
-    - vishh
-    - xiang90
+    - ahg-g
+    - alculquicondor
+    - damemi
+    - Huang-Wei
     privacy: closed
   sig-scheduling-bugs:
     description: ""
     members:
-    - aveshagarwal
-    - bsalamat
-    - ConnorDoyle
-    - davidopp
-    - derekwaynecarr
-    - gyliu513
-    - hongchaodeng
-    - ingvagabund
-    - jayunit100
-    - k82cn
-    - kevin-wangzefeng
-    - pigmej
-    - quinton-hoole
-    - ravisantoshgudimetla
-    - resouer
-    - rrati
-    - timothysc
-    - vikaschoudhary16
-    - vishh
-    - wanghaoran1988
-    - xiang90
+    - ahg-g
+    - alculquicondor
+    - chendave
+    - damemi
+    - denkensk
+    - Huang-Wei
+    - kerthcet
+    - sanposhiho
     privacy: closed
   sig-scheduling-feature-requests:
```

Comment:
Shouldn't it be just the Chairs and TLs here?

Reply:
Same as above. It's fine to "notify" more ppl about incoming feature requests.